pages/oss.tsx

```diff
@@ -0,0 +1,299 @@
+
+const contributors = [
+  {
+    name: "aeither",
+    contributions: 3,
+    image: "https://avatars.githubusercontent.com/u/36173828?s=60&v=4",
+  },
+  {
+    name: "yehia67",
+    contributions: 2,
+    image: "https://avatars.githubusercontent.com/u/21314724?s=60&v=4",
+  },
+  {
+    name: "mykcryptodev",
+    contributions: 2,
+    image: "https://avatars.githubusercontent.com/u/89474773?s=60&v=4",
+  },
+  {
+    name: "b4s36t4",
+    contributions: 2,
+    image: "https://avatars.githubusercontent.com/u/59088937?s=60&v=4",
+  },
+  {
+    name: "lucoadam",
+    contributions: 1,
+    image: "https://avatars.githubusercontent.com/u/43247609?s=60&v=4",
+  },
+  {
+    name: "JustinTime42",
+    contributions: 1,
+    image: "https://avatars.githubusercontent.com/u/5600414?s=60&v=4",
+  },
+  {
+    name: "retocrooman",
+    contributions: 1,
+    image: "https://avatars.githubusercontent.com/u/83130624?s=60&v=4",
+  },
+  {
+    name: "easonchai",
+    contributions: 1,
+    image: "https://avatars.githubusercontent.com/u/43127107?s=60&v=4",
+  },
+  {
+    name: "ElasticBottle",
+    contributions: 1,
+    image: "https://avatars.githubusercontent.com/u/44563205?s=60&v=4",
+  },
+  {
+    name: "simonlim94",
+    contributions: 1,
+    image: "https://avatars.githubusercontent.com/u/10826896?s=60&v=4",
+  },
+  {
+    name: "jnebab",
+    contributions: 1,
+    image: "https://avatars.githubusercontent.com/u/6265768?s=60&v=4",
+  },
+].sort((a, b) => b.contributions - a.contributions);
+
+const repositories = [
+  {
+    id: "contracts",
+    name: "Contracts",
+    description:
+      "Solidity smart contracts that power our ContractKit and Explore.",
+  },
+  {
+    id: "dashboard",
+    name: "Dashboard",
+    description:
+      "User interface to deploy, explore and interact with smart contracts.",
+  },
+  {
+    id: "js",
+    name: "JS Monorepo",
+    description:
+      "TypeScript, React and Wallet SDKs, CLI, Auth, Storage, and more.",
+  },
+  {
+    id: "python-sdk",
+    name: "Python SDK",
+    description: "Create web3 backend applications using Python.",
+  },
+  {
+    id: "unity-sdk",
+    name: "Unity SDK",
+    description: "Create web-based or native web3 games using Unity.",
+  },
+  {
+    id: "go-sdk",
+    name: "Go SDK",
+    description: "Create web3 backend applications using Go.",
+  },
+  {
+    id: "docs",
+    name: "Portal",
+    description: "Documentation, templates, and guides for developers.",
+  },
+  {
+    id: "solana",
+    name: "Solana SDK",
+    description: "Build web3 applications on the Solana blockchain.",
+  },
+  {
+    id: "dynamic-contracts",
+    name: "Dynamic Contracts",
+    description:
+      "Architectural pattern for writing dynamic smart contracts in Solidity.",
+  },
+];
+
+interface RepoCardProps {
+  title: string;
+  description: ReactNode;
+  url: string;
+}
+
+const RepoCard: React.FC<RepoCardProps> = ({ title, description, url }) => {
+  return (
+    <Box
```

Comment:
please use `<LinkBox>` and `<LinkOverlay>` for this (look it up in chakra docs or find it elsewhere where we already use it)

Reply:
changed to linkbox